packages/common/src/api/tan-query/comments.ts

```diff
@@ -69,8 +70,8 @@ const messages = {
  *
  */
 export type GetCommentsByUserIdArgs = {
-  userId: ID
-  currentUserId?: ID
+  userId: ID | null
+  currentUserId?: ID | null
```

Comment:
I think we've been using a pattern in other tan-query hooks of fetching this inside the hook automatically instead of making it an outer hook argument.